graylog2-web-interface/src/views/components/visualizations/GenericPlot.jsx

```diff
@@ -124,24 +124,28 @@ class GenericPlot extends React.Component<Props, State> {
       autosize: true,
       showlegend: true,
       margin: {
-        autoexpand: true,
         t: 10,
         l: 40,
         r: 10,
-        b: 40,
+        b: 0,
         pad: 0,
       },
       legend: {
         orientation: 'h',
-        y: -0.14,
       },
       hoverlabel: {
         namelength: -1,
       },
+      yaxis: {
+        automargin: true,
+      },
+      xaxis: {
+        automargin: true,
+      },
```

Comment:
As far as I could see it the `automargin` attribute is currently only helpful when using the Heatmap. I still prefer adding them to the default layout, because they could be helpful for all charts in theory,